src/main/java/duke/DukeExceptions.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+public class DukeExceptions extends Exception{
```

Comment:
While the class name is valid according to coding standards, perhaps it would be more intuitive if it was named `DukeException` instead since it only represents an instance of 1 Exception